source/apiVolontaria/volunteer/permissions.py

```diff
@@ -14,8 +14,11 @@ def has_object_permission(self, request, view, obj):
         if request.method in permissions.SAFE_METHODS:
             return True
 
-        # Write permissions are only allowed to the owner of the snippet.
-        if obj.user == request.user or request.user.is_superuser:
+        # Write permissions are only allowed to the owner of the snippet
+        # Or to a Cell Manager.
+        if obj.user == request.user or \
+                request.user.is_superuser or \
+                request.user in obj.event.cell.managers.all():
```

Comment:
It's good for me but it can be more readable to use some temp variable to help future developer :

```
is_owner = obj.user == request.user
is_admin = request.user.is_superuser
is_manager = request.user in obj.event.cell.managers.all()
can_edit = is_owner or is_admin or is_manager

if can_edit:
    ...

```

Reply:
Yes, thank for the suggestion !